src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -358,6 +361,14 @@ public override Symbol AssociatedSymbol
             }
         }
 
+        public bool LocalsAreZeroed
+        {
+            get
+            {
+                return _localsAreZeroed;
```

Comment:
>_localsAreZeroed [](start = 23, length = 16)

Why is it safe to return this value? Are we certain attributes have been bound and processed?

Reply:
Fixed.